lib/util.py

```diff
@@ -434,6 +432,8 @@ def format_satoshis(x, is_diff=False, num_zeros = 0, decimal_point = 8, whitespa
         result = " " * (15 - len(result)) + result
     return result
 
+FEE_PRECISION=1
```

Comment:
Can you add a comment to explain this? e.g. "number of digits after decimal separator"